glslc/src/file_compiler.h

```diff
@@ -75,6 +78,9 @@ class FileCompiler : public shaderc_util::Compiler {
   // Returns the name of the output file, given the input_filename string.
   std::string GetOutputFileName(std::string input_filename);
 
+  // Resoloves relative paths against this working directory.
```

Comment:
Updated the blurb to mention the invariant.

Not only relative paths of files on the command line. relative include files should also be resolved against this.